compiler/crates/relay-bin/src/main.rs

```diff
@@ -57,6 +74,28 @@ struct Opt {
     validate: bool,
 }
 
+#[derive(Parser)]
+#[clap(about = "Run the LSP server")]
+struct LspCommand {
+    /// Run the LSP using this config file. If not provided, searches for a config in
+    /// package.json under the `relay` key or `relay.config.json` files among other up
+    /// from the current working directory.
+    config: Option<PathBuf>,
+
+    #[clap(flatten)]
+    cli_config: CliConfig,
```

Comment:
I think we can omit this. I believe it's only left in the current version for backward compat, and we don't need to do that for this command.